packages/react/src/Hero/Hero.module.css

```diff
@@ -4,12 +4,40 @@
   flex-direction: column;
 }
 
+.Hero--layout-image {
+  display: grid;
+}
+
+.Hero--image-pos-inline-end {
+  grid-template-columns: 1fr;
+  gap: var(--base-size-48);
+}
+
+@media screen and (min-width: 48rem) {
+  .Hero--image-pos-inline-end {
+    align-items: center;
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+.Hero--image-pos-block-end {
+  grid-template-columns: 1fr;
+  gap: var(--base-size-40);
+}
+
+@media screen and (min-width: 48rem) {
+  .Hero--image-pos-block-end {
+    gap: var(--base-size-80);
+  }
+}
+
```

Comment:
for consistency across other elements in the page, shouldn't we use the Grid component instead?

Reply:
Fair point. I can look into this but the spacing in the site refactor designs doesn't quite align with what we have in the grid today.

Reply:
Looking at the design files it seems they've been using the Grid layout so spacing/gaps should be similar 🤔 

<img width="580" alt="Screenshot 2023-08-02 at 15 23 50" src="https://github.com/primer/brand/assets/912236/3d495a2b-b2d4-4e67-a313-b7b3ec1d19fa">

Reply:
It's the row gap that's the issue, not column...

![Screenshot 2023-08-02 at 14 35 49](https://github.com/primer/brand/assets/13340707/f815fd32-0ccf-4c3b-81e2-6e3f1b48c120)

I've switched to Grid internally, but it needed various workarounds... you can check it out [here](https://github.com/primer/brand/pull/366/commits/cba3cd691b07306ef494671cdafc1a7fc927de0f).

Reply:
nice! I see what you mean with the workarounds.. Would it make sense to use the `Box` and marginBlockStart options to manage this gap instead of the grid row gap values?

Reply:
We could do 🤔 .. I think I prefer the Grid approach you suggested before personally and switching out again feels redundant at this point. Given we want to use Box sparingly as maintainers, using the locally scoped Grid token here feels the the paved path approach to fixing this. Does that work for you too @josepmartins?

Reply:
It works! Sorry I wasn't clear, I mentioned using Box (along with the Grid) to not modify the CSS properties of the grid row gap. But locally scoped grid tokens works too oc 👍